### PR TITLE
OSDOCS-3944: Adding support for AWS Graviton3 processors

### PR DESCRIPTION
**Pull request by kelbrown20**
For versions 4.11+ 
Description: AWS Graviton3 processors are now available for OCP clusters
Jira: [OSDOCS-3944](https://issues.redhat.com//browse/OSDOCS-3944)

Previews: 

- [Installing a cluster on AWS with customizations -> Installation configuration parameters ](https://kelbrown20.github.io/openshift-docs/graviton3-on-aws-arm-support/installing/installing_aws/installing-aws-customizations.html#installation-configuration-parameters_installing-aws-customizations)
- [Installing a cluster on AWS with customizations -> Supported AWS machine types](https://kelbrown20.github.io/openshift-docs/graviton3-on-aws-arm-support/installing/installing_aws/installing-aws-customizations.html#installation-supported-aws-machine-types_installing-aws-customizations)

### CODE REVIEW

**Comment by kelbrown20** [comment]
Ref [ARMOCP-415](https://issues.redhat.com/browse/ARMOCP-415)

File: modules/installation-supported-aws-machine-types.adoc [hunk 1]
```diff
@@ -408,6 +408,11 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
 |===
 ====
 
+[IMPORTANT]
+====
+`C7g` instance types are only available on AWS Graviton3 processors.
```
Comment (yselkowitz) [inline modules/installation-supported-aws-machine-types.adoc:413]: Not quite sure of the point of this comment.  IIUC, by definition, each instance class is provided on a single CPU type.  If this is meant to indicate limited availability (as above), then let's make that explicit?

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -705,7 +705,7 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 ifndef::openshift-origin[]
 [IMPORTANT]
 ====
-When running on ARM based AWS instances, ensure that you enter a region where AWS Graviton processors are available. See link:https://aws.amazon.com/ec2/graviton/#Global_availability[Global availability] map in the AWS documentation.
+When running on ARM based AWS instances, ensure that you enter a region where AWS Graviton processors are available. Currently, AWS Graviton3 processors are only available in regions `us-east-1` and `us-west-1`. See link:https://aws.amazon.com/ec2/graviton/#Global_availability[Global availability] map in the AWS documentation.
```
Comment (yselkowitz) [inline modules/installation-configuration-parameters.adoc:708]: Making a detailed list this early on means that someone is going to have to be on top of keeping this up to date.  Maybe just say "only available in some regions" and rely on users following the link to see the current list?

File: modules/installation-configuration-parameters.adoc [hunk 3]
```diff
@@ -705,7 +705,7 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
...
 ====
```
Comment (jeffdyoung) [inline modules/installation-configuration-parameters.adoc:709]: AWS has added `us-east-2` and `eu-west-1` 

https://aws.amazon.com/about-aws/whats-new/2022/06/amazon-ec2-c7g-instances-powered-aws-graviton3-available-us-east-ohio-europe-ireland/

Comment (yselkowitz) [reply modules/installation-configuration-parameters.adoc:709]: my point exactly... :-)

File: modules/installation-supported-aws-machine-types.adoc (hunk 1 above)
Comment (kelbrown20) [reply modules/installation-supported-aws-machine-types.adoc:413]: Gotcha! That makes sense, I think I'll remove the statement in that case. Thank you!

File: modules/installation-configuration-parameters.adoc (hunk 2 above)
Comment (kelbrown20) [reply modules/installation-configuration-parameters.adoc:708]: Yeah I think thats a great idea, I only made listed it explicitly like that bc I couldn't find a link that shows the available locations for Graviton3
@yselkowitz Do you know of a link to the AWS docs that shows the Graviton3 locations?

Comment (Prashanth684) [reply modules/installation-configuration-parameters.adoc:708]: the aws cli can get that info for you: `aws ec2 describe-instance-type-offerings --filters Name=instance-type,Values=c7g.xlarge`

File: modules/installation-configuration-parameters.adoc [hunk 4]
```diff
@@ -701,11 +701,15 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 
 |`compute.aws.region`
 |The AWS region that the installation program creates compute resources in.
-|Any valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region], such as `us-east-1`.
+|Any valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region], such as `us-east-1`. You can use the AWS CLI to access the regions available based on your selected instance type. For example:
```
Comment (kelbrown20) [inline modules/installation-configuration-parameters.adoc:704]: @Prashanth684 I added the cli command, but I'm not sure if I like the formating/wording of how it is right now. But I was wondering if the command itself looked correct?

Comment (Prashanth684) [reply modules/installation-configuration-parameters.adoc:704]: yes, the command is correct - this is the sample output:

```
{
    "InstanceTypeOfferings": [
        {
            "InstanceType": "c7g.xlarge",
            "LocationType": "region",
            "Location": "us-east-2"
        }
    ]
}

```

i don't think it is necessary to add the output here and also i have an older version of the cli, so it shows only one region.

Comment (kelbrown20) [reply modules/installation-configuration-parameters.adoc:704]: Gotcha! Thank you!

**Comment by aleskandro** [comment]
lgtm

**Pull request merged**